tests/unit/utils/game-utils.spec.js

```diff
@@ -0,0 +1,72 @@
+const {
+  compareByRankThenSuit,
+  getPlayerPnumByUsername,
+  pNumIsValid,
+} = require('../../../utils/game-utils');
+
+describe('compareByRankThenSuit', () => {
+  const aceOfClubs = {
+    rank: 1,
+    suit: 0,
+  };
+  const twoOfClubs = {
+    rank: 1,
+    suit: 0,
+  };
+  const aceOfHearts = {
+    rank: 1,
+    suit: 2,
+  };
+  const jackOfHearts = {
+    rank: 11,
+    suit: 2,
+  };
```

Comment:
We could DRY this up by reusing the Card object that's defined at the bottom of tests/e2e/support/helpers.js

That already has all the cards defined using the suit and rank so then you could use `Card.ACE_OF_HEARTS` in the same way the e2e tests do. That said, we'd probably want to move the Card object into something shared by both e2e and unit tests to do so